app/assets/javascripts/discourse/app/components/post-text-selection.gjs

```diff
@@ -62,16 +63,23 @@ export default class PostTextSelection extends Component {
   });
 
   appEventsListeners = modifier(() => {
+    this.appEvents.on("topic:current-post-scrolled", this, "handleTopicScroll");
     this.appEvents.on("quote-button:quote", this, "insertQuote");
 
     return () => {
+      this.appEvents.off(
+        "topic:current-post-scrolled",
+        this,
+        "handleTopicScroll"
+      );
       this.appEvents.off("quote-button:quote", this, "insertQuote");
     };
   });
 
   willDestroy() {
     super.willDestroy(...arguments);
 
+    cancel(this.debouncedSelectionChanged);
```

Comment:
Should we only cancel on mobile as well?

Reply:
cancel is a noop when nil is given, so checking for mobile is just more useless code